### PR TITLE
Completely revamped the litmusctl workflow

### PR DESCRIPTION
**Pull request by rajdas98**
Changes
1. BCH score moved from 5 to 7
Selection_232

2. Refactored most of the functions and code to accommodate the new commands

3. Improved Code quality. Now sonarcube grade is A

4. Adding interactive and non-interactive mode

Commands
1. litmusctl config  
a) set-account  example- `litmusctl config set-account  --endpoint "https://litmus.preview.chaosnative.com" --password "" --username "raj"`
b) get-accounts example- `litmusctl config get-accounts `
c) use-account example- `litmusctl config use-account --endpoint "https://litmus.preview.chaosnative.com"  -u "raj"`
d) view example-`litmusctl config view`

2. litmusctl create 
a) agent   example- `litmusctl create agent --agent-name "raj-new-agent" --cluster-type "external" --installation-mode "cluster"  --platform-name "AWS" --sa-exists=false --ns-exists=false --project-id "" -n` 
  ##### Minimal step to create agent
  `litmusctl create agent --agent-name "new-agent" 

b) project example- `litmusctl create project --name ""`

3. litmusctl get 
a) projects example- `litmusctl get projects`
b) agents  example- `litmusctl get agents --project-id`

### CODE REVIEW

**Comment by rajdas98** [comment]
Please ignore the lint. @Jonsy13 / @arkajyotiMukherjee can you guys do a review

File: README.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ $ litmusctl version
 To connect Litmus Chaos agent:
 
 ```shell
-$ litmusctl agent connect
+$ litmusctl agent agent
```
Comment (arkajyotiMukherjee) [inline README.md:88]: Should this be `litmusctl create agent` as I understand from the new dir structure

**Review by arkajyotiMukherjee — APPROVED** [review]
Thank you for this addition. This is very good

File: README.md (hunk 1 above)
Comment (rajdas98) [reply README.md:88]: Yes, @arkajyotiMukherjee . We need to change it once we released it.

Comment (arkajyotiMukherjee) [reply README.md:88]: is `litmusctl agent agent` a valid command even for the old release.
If we mean to change this after the release then maybe we should revert the change to `litmusctl agent connect` @rajdas98

Comment (rajdas98) [reply README.md:88]: Okay, it's a typo. Sorry

Comment (rajdas98) [reply README.md]: Done @arkajyotiMukherjee

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]

File: pkg/agent/ops.go [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+}
+
+// GetMode gets mode of agent installation as input
+func GetModeType() string {
+	var mode int = 1
+	fmt.Println("\n🔌 Installation Modes:\n1. Cluster\n2. Namespace")
+	fmt.Print("\n👉 Select Mode [", utils.DefaultMode, "]: ")
+	fmt.Scanln(&mode)
+
+repeat:
```
[the remark this answers is not in the document]
Comment (rajdas98) [reply pkg/agent/ops.go]: Thanks, I didn't see that. Fixed now

File: pkg/agent/ops.go [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+	if mode == 1 {
+		return "cluster"
+	}
+
+	if mode == 2 {
+		return "namespace"
+	}
+
+	for mode < 1 || mode > 2 {
```
[the remark this answers is not in the document]
Comment (rajdas98) [reply pkg/agent/ops.go]: Done

**Review by arkajyotiMukherjee — APPROVED** [review]
Thanks for the quick changes 🚀

File: pkg/agent/ops.go [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+
+	if mode == 1 {
+		return "cluster"
+	}
+
+	if mode == 2 {
+		return "namespace"
+	}
+
+	if mode != 1 || mode != 2 {
```
Comment (arkajyotiMukherjee) [inline pkg/agent/ops.go:79]: there is a reviewdog error here...can you check that

Comment (rajdas98) [reply pkg/agent/ops.go]: fixed

**Review by arkajyotiMukherjee — APPROVED** [review]
All good 👍🏼

**Pull request merged**